### PR TITLE
feat: add `trip_id` to `vehicle_trips` tables

### PR DESCRIPTION
**Pull request by rymarczy**
This is a first attempt at adding `trip_id` to the `vehicle_trips` table.

This methodology can be refined by a future PR, but `trip_id` is required to move forward with updated `static_trip_id_guess` matching need for OPMI QA improvements.

Asana Task: https://app.asana.com/0/1204389777951216/1204505200416990
Asana Task: https://app.asana.com/0/1204517240145658/1204119636952116

### CODE REVIEW

File: python_src/tests/test_files/april_2023_filepaths.json
Comment (rymarczy) [inline python_src/tests/test_files/april_2023_filepaths.json:1]: new seed files needed to test RT vehicle events with valid `trip_id` data

File: python_src/src/lamp_py/postgres/postgres_utils.py [hunk 1]
```diff
@@ -266,16 +266,29 @@ def select_as_list(
...
-        truncat_as.create(self.engine)
+        truncate_query = f"TRUNCATE {truncat_as}"
+
+        if restart_identity:
+            truncate_query = f"{truncate_query} RESTART IDENTITY"
+
+        if cascade:
+            truncate_query = f"{truncate_query} CASCADE"
+
+        self.execute(sa.text(f"{truncate_query};"))
```
Comment (rymarczy) [inline python_src/src/lamp_py/postgres/postgres_utils.py:291]: Updated truncate function that doesn't drop and re-create whole table. Runs standard postgress truncate command.

File: python_src/src/lamp_py/performance_manager/l0_gtfs_rt_events.py [hunk 2]
```diff
@@ -108,6 +110,7 @@ def combine_events(
         "stop_sequence",
         "trip_stop_hash",
         "vehicle_id",
+        "trip_id",
```
Comment (rymarczy) [inline python_src/src/lamp_py/performance_manager/l0_gtfs_rt_events.py:113]: Pull `trip_id` from parquet files

File: python_src/src/lamp_py/performance_manager/l0_rt_trip_updates.py [hunk 3]
```diff
@@ -30,6 +30,7 @@ def get_tu_dataframe_chunks(
         "start_date",
         "start_time",
         "vehicle_id",
+        "trip_id",
```
Comment (rymarczy) [inline python_src/src/lamp_py/performance_manager/l0_rt_trip_updates.py:33]: Pull `trip_id` from parquet files

File: python_src/src/lamp_py/performance_manager/l1_rt_trips.py [hunk 4]
```diff
@@ -17,14 +18,31 @@
...
+    """
+    new_trip_hash_bytes = events.loc[
+        events["do_insert"], "trip_hash"
+    ].drop_duplicates()
+
+    db_manager.truncate_table(TempHashCompare)
+    db_manager.execute_with_data(
+        sa.insert(TempHashCompare.__table__),
+        new_trip_hash_bytes.to_frame("trip_stop_hash"),
+    )
```
Comment (rymarczy) [inline python_src/src/lamp_py/performance_manager/l1_rt_trips.py:35]: Broke up this process into a few smaller functions

File: python_src/src/lamp_py/performance_manager/l1_rt_trips.py [hunk 5]
```diff
@@ -17,14 +18,31 @@
...
+
+
+def load_new_trip_data(
+    db_manager: DatabaseManager, events: pandas.DataFrame
+) -> None:
+    """
```
Comment (rymarczy) [inline python_src/src/lamp_py/performance_manager/l1_rt_trips.py:41]: This works the same as before, just with slightly less scope in this function and the addition of `trip_id`

File: python_src/src/lamp_py/performance_manager/l1_rt_trips.py [hunk 6]
```diff
@@ -85,6 +96,54 @@ def insert_trips_hash_columns(
...
+
+    update_query = (
+        sa.update(VehicleTrips.__table__)
+        .where(
+            VehicleTrips.trip_hash == new_stop_counts_cte.c.trip_hash,
+        )
+        .values(stop_count=new_stop_counts_cte.c.stop_count)
+    )
+
+    db_manager.execute(update_query)
```
Comment (rymarczy) [inline python_src/src/lamp_py/performance_manager/l1_rt_trips.py:133]: Process updates to `stop_count` in the `vehicle_trips` table separately.

File: python_src/src/lamp_py/performance_manager/l0_rt_vehicle_positions.py [hunk 7]
```diff
@@ -136,6 +137,8 @@ def transform_vp_timestamps(
             "vehicle_id",
         ],
     )
+    # TODO: review trip_id processing # pylint: disable=fixme
+    #  add more intelligent trip_id processing, this approach will randomly select trip_id record to keep
```
Comment (rymarczy) [inline python_src/src/lamp_py/performance_manager/l0_rt_vehicle_positions.py:141]: The logic we are using to add `trip_id` is slightly random, it is functional but should be reviewed in the future to see exactly what `trip_id` we are pulling compared to the `trip_hash` grouping we are doing separately.

**Review by mzappitello — COMMENTED** [review]
a few questions before merging.

**Comment by rymarczy** [comment]
Incorporated the following comments: 
- added new seed metadata files to docker-compose.yml
- renamed `trip_stop_hash` in the `temp_hash_compare` table to `hash`

**Comment by mzappitello** [comment]
LGTM

**Pull request merged**